jaeger_client/tracer.py

```diff
@@ -237,8 +240,8 @@ def report_span(self, span):
         self.reporter.report_span(span)
         self.metrics.spans_finished(1)
 
-    def random_id(self):
-        return self.random.getrandbits(constants.MAX_ID_BITS)
+    def random_id(self, max_id_bits):
```

Comment:
* for backwards compat, please don't change this function. Instead, mark it as deprecated, and use a new private one, `_random_id(bitsize)`

Reply:
That makes sense, thank you.